assembl/static2/js/app/utils/timeline.js

```diff
@@ -24,6 +24,22 @@ const dateComparator = (phase1, phase2) => {
   return 1;
 };
 
+export const getIfDebateIsStarted = (_timeline: Timeline) => {
```

Comment:
I think that the name of this function should be `getIsDebateStarted` or just `isDebateStarted`